doc/sample/npc_test_duplicate.txt

```diff
@@ -12,7 +12,7 @@
 //=  'OnInit' loads the middle Poring last, for some reason.
 //============================================================
 
--	script	Test Script	-1,1,1,{
```

Comment:
i don't know if this should work
however it case crash for the mapserver if there is no npc object in the map for OnTouch npc